File: Zad1.cpp
#include <iostream>
using namespace std;

class Car {
public:
	string make;
	string model;
	int year;

	Car(string make, string model, int year) {
		this->make = make;
		this->model = model;
		this->year = year;
	}

	void displayInfo() {
		cout << make << " " << model << " " << year << endl;
	}
	
	~Car(){}
};

int main() {
	Car samochod1("Fiat", "500", 2017);
	samochod1.displayInfo();

	return 0;
}

File: Zad1.java
package test;
public class Car {
	
	String make;
	String model;
	int year;

	public Car(String a, String b, int c) {
		make = a;
		model = b;
		year = c;
	}
	
	void displayInfo() {
		System.out.println(make + " " + model + " " + year);
	}
	
	public static void main(String[] args) {
		Car samochod1 = new Car("Fiat", "500", 2017);
		samochod1.displayInfo();
	}
}
